shared/src/packages/journal/libs/validation-schemas/create-journal-entry.validation-schema.ts

```diff
@@ -0,0 +1,24 @@
+import joi from 'joi';
+
+import {
+  JournalEntryValidationMessage,
+  JourneyEntryValidationRule,
+} from '../enums/enums.js';
+import { type JournalEntryCreateRequestDto } from '../types/types.js';
+
+const createJournalEntry = joi.object<JournalEntryCreateRequestDto, true>({
+  title: joi
+    .string()
+    .trim()
+    .required()
+    .max(JourneyEntryValidationRule.MAXIMUM_TITLE_LENGTH)
+    .min(JourneyEntryValidationRule.MINIMUM_TITLE_LENGTH)
```

Comment:
Is there a reason to set min `1` if you already have `required`? You even didn't set message for it because there's no way it will happen)

Reply:
You are right.